examples/viewer.py

```diff
@@ -73,6 +94,59 @@ def __init__(self, sim_settings: Dict[str, Any]) -> None:
             key.Z: "move_up",
         }
 
+        # Load a TrueTypeFont plugin and open the font file
+        self.display_font = text.FontManager().load_and_instantiate("TrueTypeFont")
+        relative_path_to_font = "../data/fonts/ProggyClean.ttf"
+        self.display_font.open_file(
+            os.path.join(os.path.dirname(__file__), relative_path_to_font),
+            180,
+        )
+
+        # Glyphs we need to render everything
+        self.glyph_cache = text.GlyphCache(mn.Vector2i(2048))
```

Comment:
```suggestion
        self.glyph_cache = text.GlyphCache(mn.Vector2i(256))
```

... we discussed this on Slack some weeks ago, so just a reminder :) It'll also avoid unnecessary GPU memory use, and since the font is a bitmap one, rendering it at 10x the size will not make it any prettier.